mama_ng_control/apps/vumimessages/views.py

```diff
@@ -61,17 +61,19 @@ def post(self, request, *args, **kwargs):
                         message.metadata["ack_timestamp"] = \
                             request.data["timestamp"]
                         scheduler_ack.delay(
-                            message.metadata["scheduler_message_id"])
+                            message.metadata["subscription"])
+                        message.save()
                     elif event == "delivery_report":
                         message.delivered = True
                         message.metadata["delivery_timestamp"] = \
                             request.data["timestamp"]
+                        message.save()
```

Comment:
@imsickofmaps Okay. I was under the impression it could potentially skip the 'ack' event and just give you a 'delivery_report' event. Two follow-up questions: 
1. Why are you then setting the message.delivered = True again?
2. Can a delivery_report not have some kind of 'false' value?